packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -102,6 +108,24 @@ class EmbeddedContentConsumer extends React.Component {
       this.props.onMount(this.xfcFrame);
     }
 
+    // Cover other scenario where xfc frame style doesn't apply
+    // such as when `srcdoc` attribute is used which doesn't work
+    // within xfc's JSONRPC communication.
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('focus', () => {
```

Comment:
The contentWindow gives you a reference to the window for the embedded content.  You should be able to use document.documentElement (HTML tag) or document.body to handle the logic as necessary.

Reply:
Yeah, I'd like to handle the focus within the iframe element though, not everywhere else in the document body.

Reply:
I think you can handle by styling the iFrame element.  How are you giving focus to your iFrame?  Did you set a tabIndex?